app/views/govuk_publishing_components/components/docs/title.yml

```diff
@@ -78,4 +78,15 @@ examples:
       margin_bottom: 0
     context:
       dark_background: true
-
+  using_design_system_template:
+    description: |
+      Page title component used in a [Design System page template](https://design-system.service.gov.uk/styles/page-template/default/index.html).
+      This example is a reference for transitioning the current template used in GOV.UK to the Design System one without introducing a breaking change.
+      When all applications will be using the Design System template the component will be updated, specifying `margin_top: 0` will not be required and this example will be removed.
```

Comment:
Nitpick: these line breaks aren't honoured in the output unless there's an empty line between them, is this what you intended?

Reply:
Yeah, arbitrary (phrase-based) soft-wrapping 😁. I can move them all on one line and let editors do the soft-wrap.

Reply:
Even better, I replaced the content with your suggestion 👍